### PR TITLE
enum type fix

### PR DESCRIPTION
**Pull request by tommydcjung**

### CODE REVIEW

File: hardware/axil_to_mcl.v [hunk 1]
```diff
@@ -394,7 +394,7 @@ module axil_to_mcl
     assign fifo_req_cast[i]          = fifo_data_lo[2*i];
 
     bsg_manycore_packet_op_e endpoint_out_packet_op;
-    assign endpoint_out_packet_op = fifo_req_cast[i].op;
+    assign endpoint_out_packet_op = bsg_manycore_packet_op_e'(fifo_req_cast[i].op);
```
Comment (mrutt92) [inline hardware/axil_to_mcl.v]: why is this necessary?

Comment (drichmond) [reply hardware/axil_to_mcl.v]: Vivado synthesis fails with the following error

```ERROR: [Synth 8-1587] an enum variable may only be assigned to same enum typed variable or one of its values [/mnt/users/spin0/no_backup/drichmond/Research/repositories/git/bsg_bladerunner.bak/bsg_f1/build/src_post_encryption/axil_to_mcl.v:397]```

Comment (mrutt92) [reply hardware/axil_to_mcl.v]: got it. thanks!

Comment (tommydcjung) [reply hardware/axil_to_mcl.v]: This also causes lint warning in vcs. We should do our best to pay attention to those warnings and fix them.
